test/selenium_tests/test_library_to_collections.py

```diff
@@ -1,17 +1,17 @@
-from base.populators import flakey
+import unittest
 
 from .framework import (
     selenium_test,
     SeleniumTestCase,
 )
 
 
+@unittest.SkipTest
```

Comment:
I think this should be [@unittest.skip()](https://docs.python.org/2/library/unittest.html#unittest.skip).

Reply:
Fixed, that was silly of me.